Source/CarShack/Hypermedia/Hypermedia.Server.g.cs

```diff
@@ -0,0 +1,212 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bluehands.Hypermedia.Relations;
+using WebApi.HypermediaExtensions.Hypermedia;
+using WebApi.HypermediaExtensions.Hypermedia.Actions;
+using WebApi.HypermediaExtensions.Hypermedia.Attributes;
+using WebApi.HypermediaExtensions.Hypermedia.Extensions;
+using WebApi.HypermediaExtensions.Hypermedia.Links;
+using WebApi.HypermediaExtensions.Query;
+using WebApi.HypermediaExtensions.WebApi.RouteResolver;
+
+namespace CarShack.Hypermedia;
+
+public static class MimeTypes
+{
+    public const string APPLICATION_VND_SIREN_JSON = "application/vnd.siren+json";
```

Comment:
Can we use Bluehands.Hypermedia.MediaTypes.DefaultMediaTypes.Siren?

Reply:
I'd rather have the literal string here as to not force a user to add a reference. According to @AydinMirMohammadi the project should be removed entirely and the mime types should be copied if needed.